.github/workflows/ci_issues_triage.yml

```diff
@@ -52,7 +55,10 @@ jobs:
         labeled: VST
 
   assign:
-    if: github.event.action == 'labeled' || github.event.action == 'unlabeled'
+    if: |
+      (github.event.action == 'labeled' || github.event.action == 'unlabeled')
+      && github.event.issue.state == 'open'
+      && !contains( github.event.issue.labels.*.name, 'disable-auto-assign')

```

Comment:
please explain how it works

Reply:
Sometimes, you want to add a label but not automatically assign someone. The idea is that you can first add a new 'disable-auto-assign' label, then add the label you want to add, and finally remove the 'disable-auto-assign' label. 

This line just checks whether the 'disable-auto-assign' label is present (and whether the issue is currently open).

Reply:
Can you give examples of issues where this caused problems?
It just seems to me that this is a workaround and not a real solution

Reply:
It's not necessarily a solution or workaround, but actually a feature. For example, it is useful for old issues: often, you do want to give them the correct label, but you don't want to disturb people by automatically assigning them. 
Or, as @oktophonie mentioned, sometimes for closed issues you do want to add the label (just for the record), but not assign people and also not make the issue automatically appear in a project.

So, most of the times, you do want "auto-assign", but sometimes you don't want it. This PR adds the possibility to do that.

If you have a better idea for how we can disable "auto-assign" for single cases, that's welcome though.

Reply:
I agree with not auto-assigning to closed issues. It is not obvious in what cases we need to disable auto-assign for opened issues

Reply:
Okay, I removed it. Then we simply have to accept the fact that we'll sometimes be annoyed with "musebot assigned you" notifications for old issues when someone is adding missing labels.